.gitignore

```diff
@@ -2,3 +2,7 @@ config.yml
 __pycache__
 /engines/*
 !/engines/README.md
+
+# avoid pushing project files when people use PyCharm
+.idea/
```

Comment:
No. Your environment specific excludes must go to your own global gitignore file.